packages/api/src/services/WalletService.ts

```diff
@@ -245,6 +245,7 @@ export default class Wallet extends Service {
     validateOnly?: boolean;
     disableJSONFormatting?: boolean;
     maxTime?: number;
+    offerOnly?: boolean;
   }) {
     const { disableJSONFormatting, driverDict, ...restArgs } = args;
     return this.command<{ offer: string; tradeRecord: TradeRecord }>(
```

Comment:
When offer_only flag is true this should only return the offer and not the traderecord

Reply:
OK, I'll check.